src/user_templates_api/templates/jupyter_lab/templates/hra_ui_widgets_tutorial/template.txt

```diff
@@ -0,0 +1,194 @@
+[
+  {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+      "## Usage\n",
+      "This notebook shows how to use the different widgets provided by the hra_jupyter_widgets package."
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "# linked datasets\n",
+      "uuids = {{ uuids | safe }}"
+    ]
+  },
+  {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+      "### Install package"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "%pip install hra_jupyter_widgets"
+    ]
+  },
+  {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+      "### Import widgets"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "import ipywidgets as widgets\n",
+      "from hra_jupyter_widgets import (\n",
+      "    EuiOrganInformation,\n",
+      "    FtuExplorer,\n",
+      "    FtuExplorerSmall,\n",
+      "    MedicalIllustration,\n",
+      "    ModelViewer\n",
+      ")"
+    ]
+  },
+  {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+      "### Configure organ and illustration"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "# Kidney uberon id\n",
+      "organ = \"http://purl.obolibrary.org/obo/UBERON_0002113\"\n",
+      "illustration = \"https://purl.humanatlas.io/2d-ftu/kidney-renal-corpuscle\""
+    ]
+  },
+  {
+    "cell_type": "markdown",
+    "metadata": {},
+    "source": [
+      "### Create and use a widget"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "medical_illustration = MedicalIllustration(selected_illustration=illustration)\n",
+      "display(medical_illustration)"
+    ]
+  },
+  {
+    "cell_type": "code",
+    "execution_count": null,
+    "metadata": {},
+    "outputs": [],
+    "source": [
+      "# Create an output widget\n",
+      "output = widgets.Output()\n",
+      "\n",
+      "\n",
+      "# Create an event listener that prints to the output widget\n",
+      "# It takes a single argument - the event data\n",
+      "def cell_click(cell):\n",
+      "    with output:\n",
+      "        print(f\"Cell clicked: '{cell['label']}'\")\n",
+      "\n",
+      "\n",
+      "# Register the listener\n",
+      "medical_illustration.on_cell_click(cell_click)\n",
+      "# Listeners can be removed by calling on_cell_click with remove=True, i.e.\n",
+      "# medical_illustration.on_cell_click(cell_click, remove=True)\n",
+      "\n",
+      "display(output)"
+    ]
+  },
```

Comment:
What does this cell do?

Reply:
When they click on a cell, it will display it in the notebook. Theoretically you could use that call back to do other interesting things with the cell type it returns.